redaxo/src/addons/structure/CHANGELOG.md

```diff
@@ -1,6 +1,14 @@
 Changelog
 =========
 
+Version 2.11.0 – XX.XX.2020
+---------------------------
+
+### Neu
+
+* Fragment `module_select.php`: Module-Key wird mit übergeben (@skerbis)
```

Comment:
Sollten wir hier ein beispiel use-case bennen?

Reply:
Ist immer die Frage, wie ausführlich man im Changelog sein möchte. Beispiele haben wir da aber eigentlich sehr selten. Wüsste jetzt auch gar nicht, was man hier für ein Beispiel anführen würde.
Ich denke, wer das Feature benötigt, wird die Zeile verstehen, allen anderen müsste man vermutlich erstmal überhaupt das Feature `module_select.php` erläutern.

Reply:
Da ich selbst mit dem text nix angangen kann hab ich die frage gestellt. 

Ggf schreib auch jemand das konkrete Beispiel in die Doku, oder als FOR trick

@skerbis meinung dazu?

Reply:
Der Key hatte dort ja nur gefehlt. 😀 - aber sicher werde ich das in der Doku erwähnen